code/api/src/seeders/4-userproducts.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const bcrypt = require('bcrypt');
+const config = require('../config/server.json');
+const params = require('../config/params.json');
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('userproducts', [
+      {
+        userId: 2,
```

Comment:
This is helpful to see and visualize. I appreciate the clarity the API contract has, but this helps even more.